tempto-core/src/test/groovy/io/prestosql/tempto/internal/initialization/TestInitializationListenerTest.groovy

```diff
@@ -20,6 +20,7 @@ import io.prestosql.tempto.context.State
 import io.prestosql.tempto.context.TestContext
 import io.prestosql.tempto.context.TestContextCloseCallback
 import io.prestosql.tempto.fulfillment.RequirementFulfiller
+import io.prestosql.tempto.fulfillment.RequirementFulfiller.TestLevelFulfiller
 import io.prestosql.tempto.fulfillment.TestStatus
 import io.prestosql.tempto.internal.TestSpecificRequirementsResolver
 import org.testng.*
```

Comment:
can you please remove `*`?

Reply:
preexisting; intellij autoformat doesn't "fix" this.
ideally i'd rather see all the groovy gone